docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -10,8 +10,11 @@ title: StatefulSets
 ---
 
 {% capture overview %}
-**StatefulSet is the workload API object used to manage stateful applications.
-StatefulSets are beta in 1.8.**
+
+StatefulSet is the workload API object used to manage stateful applications.
+
+**Note:** StatefulSets are stable (GA) in 1.9.
```

Comment:
nit: 1.9 and above?

Reply:
On second thought, seems fine even as is. Thinking from a perspective of a 1.9+ user, it should still be clear.